easybuild/easyconfigs/p/PyStan/PyStan-2.19.0.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'PyStan'
+version = '2.19.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/stan-dev/pystan'
+description = """Python interface to Stan, a package for Bayesian inference
+ using the No-U-Turn sampler, a variant of Hamiltonian Monte Carlo."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b85301b960d5991918b40bd64a4e9321813657a9fc028e0f39edce7220a309eb']
+
+dependencies = [
+    ('Python', '3.6.4'),
+]
+
+download_dep_fail = True
+
+use_pip = True
+
+options = {'modulename': '%(namelower)s'}
```

Comment:
@SimonPinches the default modulename is already the lowercase name, so you can remove this line.

Reply:
Thanks @smoors, I just (force) pushed an update to correct this.